sdks/python/container/py2/build.gradle

```diff
@@ -51,8 +52,13 @@ task copyLauncherDependencies(type: Copy) {
   }
 }
 
+def sdk_version = '2.16.0'
```

Comment:
This is probably not a good idea. The reason is:
- This needs to match the https://github.com/apache/beam/blob/master/sdks/python/apache_beam/version.py - So it needs to be 2.16.0.dev here
- Release process needs to be updated to change these versions along with other versions. Release process automatically changes the version in the release branch to remove the ".dev" part and changes master branch to the next version.

- If we have to do this, at least is it possible to put this in a single gradle file?
- Another idea, we can execute or parse https://github.com/apache/beam/blob/master/sdks/python/apache_beam/version.py and extract version from here. That should be possible with gradle.

@yifanzou could you help with this?
/cc @markflyhigh as the release manager.

Reply:
It would also be good to retain the distinction between -SNAPSHOT/.dev version and release version in the container tag. It can lead to confusing results when a locally built container from snapshot source overrides the image of a release.

Reply:
https://docs.gradle.org/current/dsl/org.gradle.api.resources.TextResource.html

Reply:
Thanks for reviewing and comments.
I wanted to make `sdk_version` consistent with the one from `version.py`, forgot to change it back after testing. Thanks for catching it.

After evaluating many options, I think introducing a new variable `sdk_version` to `gradle.properties` is the easiest way for now. This version will be consistent to the one from `version.py`, so we need to update it when we release it. It would be better to read it from `version.py` and assign to `sdk_version`, however, as far as I know, `gradle.properties` is not supposed to read variables from another file. If it's possible, I am happy to try.

I made a commit with this change, can you please review and make comments?

Thanks,
Hannah